### ISSUE TITLE
Removing sitebricks dependency

### ISSUE THREAD

**Issue by Murali54** [browserup/browserup-proxy#297]
browserup-proxy currently has a dependency on https://github.com/dhanji/sitebricks. This seems to be an abandoned project and has no activity for the last 5 years. Are there any plans of using other alternative solutions?

`sitebricks` has security vulnerabilities https://github.com/dhanji/sitebricks/pull/90

**Comment by ericbeland** [comment]
Yes, that's a goal, however we have business priorities that come before it, so it may be a bit. We welcome pull requests.

**Comment by Murali54** [comment]
Thanks, Eric for the quick response! I can definitely give it a try. Please advice.

**Comment by ericbeland** [comment]
We've started down the road of deprecating sitebricks.  There are still usages in the part of the proxy that is used to control proxy resources.  https://github.com/browserup/browserup-proxy/search?q=sitebricks&unscoped_q=sitebricks

In our new assertions API, we are working from swagger and following a pattern that looks more like this for our API:

https://github.com/browserup/browserup-proxy/blob/889aeda6d27b05b50714b754f6e43b3a600e6d9b/browserup-proxy-rest/src/main/java/com/browserup/bup/rest/resource/entries/EntriesProxyResource.java

We are going to switch our interface to a swaggerized implementation, as then we can generate REST clients for any different language.

@kiturutin Can you weigh in with any additional instructions to get setup. 

Note that we now have a dependency on Python and a forked version of mitmproxy. This will let us solve a number of bugs in the proxy and remove our littleproxy dependency.
